resources/todomvc/architecture-examples/angular/package.json

```diff
@@ -10,7 +10,7 @@
     "scripts": {
         "ng": "ng",
         "dev": "ng serve",
-        "build": "ng build --base-href /resources/todomvc/architecture-examples/angular/dist/",
+        "build": "ng build --source-map=true",
```

Comment:
we should keep the `base-href` in there.

Reply:
Do you know what this does? It's not clear to me. If all it does is adding the `<Base>` to the html file, then maybe we don't need it after all?

Reply:
I know Angular needs to be able to find the files when running inside Speedometer, but @rniwa also pointed out before that there was an issue when using `<base>` in the html file when running on the webkit infra.

Reply:
At least this works without it currently (I made sure to try it locally), so it's probably not needed unless I'm missing something.

Reply:
I remember this requiring manual fixups like #161 in the past (from @rniwa). I can't tell if we've fixed this with this config in the meantime or if it's still requiring manual work for each build (and if so we should, at minimum, document that)

Reply:
Getting rid of `--base-href` gets rid of the base element in the build so I don't think we want to keep that. As far as I can tell, there is no other side effect from this change.